### PR TITLE
Fixes from phil's testing for various things

### PR DESCRIPTION
**Pull request by ph1l**
* serve static files from apache
* fix initialize db scripts
* increase autocomplete option limit
* fix backupdb and restoredb commands' argument parsing and printing

### CODE REVIEW

File: util/initialize_test_db.sh [hunk 1]
```diff
@@ -17,7 +20,7 @@ if [ -e ${VAR}/db/ocemr.db ]; then
 fi
 
 
-python ${APP}/manage.py migrate
+python ./manage.py migrate
```
Comment (patfreeman) [inline util/initialize_test_db.sh]: A relative path like this is ugly. Why not keep it based on the variable APP?

Comment (ph1l) [reply util/initialize_test_db.sh]: I will address that, and clean up my branch a bit

**Comment by ph1l** [comment]
re: jQuery changes

I'm sure there are bugfixes in jQuery, we could definitely entertain that Idea..

**Comment by ph1l** [comment]
@patfreeman I think this is cleaned up

**Pull request merged**